control/server.py

```diff
@@ -147,6 +159,14 @@ def _add_server_listener(self):
             self.server.add_insecure_port("{}:{}".format(
                 gateway_addr, gateway_port))
 
+    def _exception_spdk(self):
+        timeout = self.config.getfloat("spdk", "timeout")
+        stdout, stderr = self.spdk_process.communicate(timeout=timeout)
+        return_code = self.spdk_process.returncode
+        self.logger.error(f"SPDK return code: {return_code}")
```

Comment:
I would suggest "exit code" (Python's terminology is unfortunate here):
```suggestion
        self.logger.error(f"SPDK exit code: {return_code}")
```